README.md

```diff
@@ -1,22 +1,29 @@
-# EvalAI-CLI
+# EvalAI CLI
 
-Official Command Line utility to use EvalAI in your terminal. This package offers almost all of the full suite of features available on the website withing your terminal.
+Official Command Line utility to use EvalAI in your terminal. 
 
-## Setting up EvalAI-CLI
+This package offers almost all of the full suite of features available on the website within your terminal.
 
-Setting up Virtual Environment.
+## Development Setup
 
-```
+1. Clone the repository to you machine
```

Comment:
Please change `you` to `your` and I think we can also add the `git clone` command for the easiness of the contributor.
`git clone https://github.com/Cloud-CV/evalai-cli.git EvalAI-CLI`